### DISCUSSION TITLE
gleam update --latest, which ignoring semver

### DISCUSSION THREAD

**Discussion by darky** [gleam-lang/gleam#3053] [Ideas & suggestions]
Now `gleam update` works with considering of semver
For example, if package version is `0.1`, updating will be in range of `0.x`, but not in `1.x`, `2.x` etc

But often, need to upgrade all dependencies to latest possible versions.
Something like `gleam update --latest` would be helpful here

And no annoying manual investigation of latest version for each dependency

**Comment by lpil** [comment]
Hello! Why would you not relax your version requirements in this case?

**Comment by darky** [comment]
Hello, thanks! Really, I can use something like this `>= 0.X.0 and < 99.0.0` without overcomplicating things 😄

**Reply by lpil** [reply]
You could even just leave off the `and x.y.z`!
